lnbits/core/crud.py

```diff
@@ -276,6 +273,9 @@ async def get_wallet(
         (wallet_id,),
     )
 
+    if row["deleted"]:
+        return None
+
```

Comment:
do this in the sql query ```WHERE id = ? and deleted = false

Reply:
deleted doesn't have a default `false`... if it's not `true` it's just `Null`, does it work like that?

Reply:
ok, just on the safe side: `WHERE id = ? AND deleted is NULL or deleted = false`